TCAT/Views/DatepickerView.swift

```diff
@@ -31,20 +32,29 @@ class DatepickerView: UIView {
     let spaceBtButtonAndSuperviewSide: CGFloat = 12.0
     let spaceBtButtonAndSegmentedControl: CGFloat = 16.0
     let spaceBtSegmentControlAndDatePicker: CGFloat = 8.0
+    let spaceBtSegmentControls: CGFloat = 8.0
     
     // MARK: Init
     
     override init(frame: CGRect){
         super.init(frame: frame)
         
         styleDatepicker()
-        styleSegmentedControl()
+        styleSegmentedControl(timeTypeSegmentedControl)
+        styleSegmentedControl(leaveNowSegmentedControl)
         styleCancelButton()
         styleDoneButton()
-        styleDisclaimerLabel()
+//        styleDisclaimerLabel()
 
         setDatepickerSettings()
-        setSegmentedControl(withItems: segmentedControlOptions)
+        
+        setSegmentedControl(timeTypeSegmentedControl, withItems: timeTypeSegmentedControlOptions)
+        let leaveAt = 0
```

Comment:
if you don't use this again, might just want to set the next line to be 0 and add a comment